hassio/src/addon-view/info/hassio-addon-info.ts

```diff
@@ -171,16 +171,16 @@ class HassioAddonInfo extends LitElement {
                   : ""}
               </div>
               <div class="card-actions">
-                <mwc-button @click=${this._updateClicked}>
-                  ${this.supervisor.localize("common.update")}
-                </mwc-button>
                 ${this.addon.changelog
                   ? html`
                       <mwc-button @click=${this._openChangelog}>
                         ${this.supervisor.localize("addon.dashboard.changelog")}
                       </mwc-button>
                     `
                   : ""}
```

Comment:
It will not be right-aligned now when there is no changelog available
```suggestion
                  : html`<span></span>`}
```